release/utils/flags-config.ts

```diff
@@ -165,13 +297,35 @@ export const command_config: CommandConfig = {
     alt: Array.from(ABOUT),
     example: '$ ./publish/index.ts about',
   },
+  release_notes: {
+    name: 'Release Notes',
+    cmd: 'release-notes',
+    alt: ['cl', 'changes', 'history', 'notes', 'releasenotes', 'changelog', 'log'],
+    description: `Generate release notes for the next release.`,
+    options: release_notes_flags_config,
+    example: '$ ./publish/index.ts release-notes',
```

Comment:
I understand almost nothing in this PR 😓 , but I guess here the example should use something like `bun release release-notes` ?